src/functions/putPassCertificateDuplicates/framework/handler.ts

```diff
@@ -0,0 +1,22 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import Response from '../../../common/application/api/Response';
+import createResponse from '../../../common/application/utils/createResponse';
+import { HttpStatus } from '../../../common/application/api/HttpStatus';
+import { bootstrapConfig } from '../../../common/framework/config/config';
+import { bootstrapLogging, error } from '@dvsa/mes-microservice-common/application/utils/logger';
+import { identifyDuplicateCertificates } from './repository/duplicate-certificate-service';
+
+export async function handler(event: APIGatewayProxyEvent): Promise<Response> {
+
+  try {
+    bootstrapLogging('identify duplicate pass certificates', event);
+    await bootstrapConfig();
+
+    const response = await identifyDuplicateCertificates();
+
+    return createResponse(response, HttpStatus.CREATED);
```

Comment:
I can't imagine we would want to return the response, in other put's we would return an empty object to indicate success.

What would `identifyDuplicateCertificates` return? Is it all of the rows that would be updated?

Reply:
hmmm....it rejects the record if the certificate number already exists.  I want to confirm this that uses the 'created' status.

will take your comment on board and refactor